proglearn/deciders.py

```diff
@@ -17,8 +17,10 @@
 
 from sklearn.utils.multiclass import type_of_target
 
+from sklearn.base import BaseEstimator, ClassifierMixin
 
-class SimpleArgmaxAverage(BaseClassificationDecider):
+
+class SimpleArgmaxAverage(BaseClassificationDecider, BaseEstimator, ClassifierMixin):
```

Comment:
You don't need to interit from BaseEstimator or ClassifierMixin. By nature of inserting from BaseClassificationDecider, it inherits from BaseDeicder, which inerits from BaseEstimator. Also, if you make BaseClassificationDecider from ClassifierMixin, then SimpleArgmaxAverage will inherit from ClassifierMixin, since it inherits from BaseClassificationDecider.